service/cloudconfig/v_0_1_0_master_template.go

```diff
@@ -1,123 +1,153 @@
-package configmap
+package cloudconfig
 
 import (
 	"github.com/giantswarm/certificatetpr"
-	k8scloudconfig "github.com/giantswarm/k8scloudconfig"
+	clustertprspec "github.com/giantswarm/clustertpr/spec"
+	"github.com/giantswarm/k8scloudconfig/v_0_1_0"
```

Comment:
Can we alias that to `k8scloudconfig` to avoid changing a lot of code in case of update or copy/pasting?

Reply:
We can do this. Good idea.

Reply:
Done.